R/pam-tidiers.R

```diff
@@ -0,0 +1,91 @@
+' Tidying methods for pam objects
+#' 
+#' These methods summarize the results of Partitioning Around Medoids (PAM) clustering into three
+#' tidy forms. \code{tidy} describes the medoid and size of each cluster,
```

Comment:
`broom` uses rmarkdown for documentation. Instead of `\code{name}` use backticks. More details are available at:

https://cran.r-project.org/web/packages/roxygen2/vignettes/markdown.html